spec/facility_factory_spec.rb

```diff
@@ -0,0 +1,169 @@
+  
+      # mo_facility_sample_1 = @missouri_facilities_data[0]
+      # expect(@missouri.format_facility_name(mo_facility_sample_1)).to eq("Dexter")
+      
+      # mo_facility_sample_2 = @missouri_facilities_data[1]
+      # expect(@missouri.format_facility_name(mo_facility_sample_2)).to eq("Clayton")
+    end
+  end
+  
+  describe "#format_facility_address" do 
+    it "formats CO facilities addresses" do 
+      random_co_facility_sample = @colorado_facilities_data.sample
+      expect(@colorado.format_facility_address(random_co_facility_sample)).to be_a String
+      
+      # co_facility_sample_1 = @colorado_facilities_data[1]
+      # expect(@colorado.format_facility_address(co_facility_sample_1)).to eq("4685 Peoria Street Suite 101 Denver CO 80239")
+      
+      # co_facility_sample_2 = @colorado_facilities_data[2]
+      # expect(@colorado.format_facility_address(co_facility_sample_2)).to eq("3698 W. 44th Avenue Denver CO 80211")
+    end
+    
+    it "formats NY facility addresses" do 
+      random_ny_facility_sample = @new_york_facilities_data.sample
+      expect(@new_york.format_facility_address(random_ny_facility_sample)).to be_a String
+      
+      # ny_facility_sample_1 = @new_york_facilities_data[0]
+      # expect(@new_york.format_facility_address(ny_facility_sample_1)).to eq("168-46 91st Ave., 2nd Flr Jamaica NY 11432")
+
+      # ny_facility_sample_2 = @new_york_facilities_data[1]
+      # expect(@new_york.format_facility_address(ny_facility_sample_2)).to eq("200 Old Country Road Riverhead NY 11901")
+    end
+    
+    it "formats MO facility addresses" do 
+      random_mo_facility_sample = @missouri_facilities_data.sample
+      expect(@missouri.format_facility_address(random_mo_facility_sample)).to be_a String
+    
+      # mo_facility_sample_1 = @missouri_facilities_data[0]
+      # expect(@missouri.format_facility_address(mo_facility_sample_1)).to eq("119 Vine St Dexter MO 63841")
+      
+      # mo_facility_sample_2 = @missouri_facilities_data[1]
+      # expect(@missouri.format_facility_address(mo_facility_sample_2)).to eq("141 N Meramec Ave, Ste 201 Clayton MO 63105")
+    end
+  end 
+
+  describe "#format_facility_phone" do 
+    it "formats CO facilities phone number" do 
+      random_co_facility_sample = @colorado_facilities_data.sample
+      expect(@colorado.format_facility_phone(random_co_facility_sample)).to be_a String
+      
+      # co_facility_sample_1 = @colorado_facilities_data[1]
+      # expect(@colorado.format_facility_phone(co_facility_sample_1)).to eq("(720) 865-4600")
+      
+      # co_facility_sample_2 = @colorado_facilities_data[2]
+      # expect(@colorado.format_facility_phone(co_facility_sample_2)).to eq("(720) 865-4600")
+    end
+
+    it "formats NY facilities phone number" do 
+      random_ny_facility_sample = @new_york_facilities_data.sample
+      expect(@new_york.format_facility_phone(random_ny_facility_sample)).to be_a String
```

Comment:
For all the phone numbers especially I would recommending testing for a specific example to make sure your method is putting the `-` and other formatting where you expect it to go.
Both "1234567890" and "123-456-7890" would pass your test as a "string" so it doesn't tell you much about what your formatter is doing.
You don't necessarily have to copy an entire hash to use as your variable, you could put in something like 
``` ruby
random_phone_number = {"phone":"3545344600"}
expect(facilityfactory.format_facility_phone(random_phone_number).to eq("(354) 534-4600")
```

Reply:
it's also way more complicated than you would need for this project,
But there is a way to test based on formatting. IE, instead of testing for the exact numbers, you can test that the result contains the format of `(`, `3 digits`, `)`, `3 digits`, `-`, `4 digits`
[It's called regex](https://www.rubyguides.com/2015/06/ruby-regex/)